experiments/AMIP/modular/README.md

```diff
@@ -84,34 +84,34 @@ https://climate.ucdavis.edu/pubs/UMJS2013QJRMS.pdf
 # Heat Slab
 The slab solves for temperature in a single layer, whose tendency is the accumulated fluxes divided by the coupling timestep plus a parameterisation of the internal processes, $G$.
 $$
-\rho c h_s  \, \partial_t T_{sfc} =  - F_{integ}  / \Delta t_{coupler}  
+\rho c h_s  \, \partial_t T_{sfc} =  - F_{integ}  / \Delta t_{coupler}
 $$
 
 # Distribution
 ## Julia multithreading
-- we can run using multiple threads if the command line argument `enable_threading` is true (e.g., run with `julia --project --threads 8`). 
+- we can run using multiple threads if the command line argument `--device` is set to `CPUMultiThreaded` and the number of threads `n` is specified with `--threads n`.
 
-## MPI via ClimaComms.jl 
+## MPI via ClimaComms.jl
 - for AMIP we want the surface columns to be on the same processors as the atmos columns. Since all all surface domains will be on masked spheres, all of them can inherit the same distributed horizontal space from the atmos model.
-- for this we need to run the `sbatch_job.sh` script, which sets up the `CLIMACORE_DISTRIBUTED` environment variable and job specifications, and runs the coupler_driver with `mpiexec` 
+- for this we need to run the `sbatch_job.sh` script, which sets up the `CLIMACORE_DISTRIBUTED` environment variable and job specifications, and runs the coupler_driver with `mpiexec`
```

Comment:
ClimaComms uses `CLIMACOMMS_CONTEXT` for the environment variable, I'm not sure that `CLIMACORE_DISTRIBUTED` is still used